src/cocoa/toga_cocoa/widgets/internal/cells.py

```diff
@@ -1,9 +1,61 @@
-from rubicon.objc import *
 
 from toga_cocoa.libs import *
 
 
+class TogaIconView(NSTableCellView):
+
+    @objc_method
+    def initWithFrame_(self, frame: CGRect):
+        self = ObjCInstance(send_super(__class__, self, 'initWithFrame:', frame))
+        return self.setup()
+
+    @objc_method
+    def init(self):
+        self = ObjCInstance(send_super(__class__, self, 'init'))
+        return self.setup()
+
+    @objc_method
+    def setup(self):
+        iv = NSImageView.alloc().initWithFrame(NSMakeRect(0, 0, 16, 16))
+        tf = NSTextField.alloc().initWithFrame(NSMakeRect(0, 0, 2000, 16))
```

Comment:
Is there any significance to the 16 and 2000 magic values here? Will they be affected by users that have increased their system default text size?

Reply:
I just copied the behaviour from the previous `TogaIconCell` which hard-coded the font size to 13 and the icon height to 16 px. But hard-coding those values does mean that larger text will be clipped.

If you want to allow setting the font of a TreeView, this should be adjusted. It will also require telling the NSOutlineView that the row heights have changed after setting a font size.

Reply:
Alright, I have removed the hard-coded frame. Now, when an image is added the textfield's origin is offset without setting the size.